### ISSUE TITLE
Plasma statues can be set off by laser tag guns

### ISSUE THREAD

**Issue by DarkPyrolord** [ParadiseSS13/Paradise#7234]
**Problem Description**:
Plasma statues are being set on fire due to laser tag guns.

**What did you expect to happen**: 
Laser tag gun misfire, leading to it hitting a statue that was behind the target, doing nothing.

**What happened instead**:
Laser tag beam hits statue, statue catches fire, death and fire for all!

**Why is this bad/What are the consequences:**
As laser tag guns are shot around often, and plasma statues are usually placed near main halls, I can see plenty of situations where someone triggers this by accident and gets a mis-bwoink.

**Steps to reproduce the problem**:
-Get a laser tag gun.
-Get a plasma statue.
-Shoot the plasma statue and watch it all burn.

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was
anything else out of the ordinary happening?)**: Noticed this over a few rounds, no real special conditions to it.

**Comment by KasparoVy** [comment]
LMAO, that's interesting.

**Comment by Anticept** [comment]
Lesson kids:

don't make plasma statues.

**Comment by taukausanake** [comment]
@Anticept you mean ALWAYS make plasma statues

**Comment by Fox-McCloud** [comment]
@uraniummeltdown

**Comment by FalseIncarnate** [comment]
Lasertag beams are a subtype of `/obj/item/projectile/beam`, and the statue bullet_act simply does a check of `istype(/obj/item/projectile/beam)` which the lastertag beam would fulfill. Easy to just adjust the check to be `if(istype(/obj/item/projectile/beam) && !istype(/obj/item/projectile/beam/lasertag))` so all other energy beam "projectiles" will set it off as intended but laser tag ones are harmless to them.

**Comment by taukausanake** [comment]
Isn't that a textbook definition of _snowflaaaake_? I mean it's the easiest way to fix it, but definitions

**Comment by Fox-McCloud** [comment]
@FalseIncarnate 

Or you could, you know, check for if the projectile has `nodamage` or not.

**Comment by deep-fried-faceplant** [comment]
@Fox-McCloud Disablers would need to get the `nodamage` var too then, unless disablers are powerful enough to ignite plasma.

**Comment by MarcellusPye** [comment]
I think this is a case where if you don't want ignited highly volitile materials in the halls, don't put highly volitile materals in the halls then play with lazers in the halls.

TLDR: No change needed
